### PR TITLE
docs: added uniform id from apisix doc

### PR DESCRIPTION
**Pull request by iamayushdas**
### What this PR does / why we need it:
uniform/particular id given to the documentation

issue: https://github.com/apache/apisix-website/issues/363

### Pre-submission checklist:

* [ ] Did you explain what problem does this PR solve? Or what new features have been added?
* [ ] Have you added corresponding test cases?
* [ ] Have you modified the corresponding document?
* [ ] Is this PR backward compatible? **If it is not backward compatible, please discuss on the [mailing list](https://github.com/apache/apisix/tree/master#community) first**

### CODE REVIEW

File: docs/en/latest/getting-started.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ title: Getting Started
 #
 -->
 
-## Getting Started
+## Getting Started {#apisix}
```
Comment (spacewander) [inline docs/en/latest/getting-started.md:24]: This breaks the render in the GitHub markdown preview (maybe also the cross reference). Can we add the mark into '' and take it away from the title?

**Review by spacewander — CHANGES_REQUESTED** [review]

File: docs/en/latest/getting-started.md (hunk 1 above)
Comment (iamayushdas) [reply docs/en/latest/getting-started.md:24]: No we can't do that, and i have tested this on my system it works fine

Comment (iamayushdas) [reply docs/en/latest/getting-started.md:24]: its not the title , the title is above the license text

Comment (juzhiyuan) [reply docs/en/latest/getting-started.md:24]: @iamayushdas I suggest submitting PRs `on one repo` for test & preview, let's have a preview here https://github.com/apache/apisix-dashboard/pull/1951 first.

Comment (iamayushdas) [reply docs/en/latest/getting-started.md:24]: okay

Comment (spacewander) [reply docs/en/latest/getting-started.md:24]: It is **the title** from the view of the markdown version. See https://github.com/apache/apisix/blob/master/docs/en/latest/getting-started.md

Comment (spacewander) [reply docs/en/latest/getting-started.md:24]: I can't accept such a scar on the title. The solution is quite a hack. How can a mark be put into the content, especially the content is obvious to everyone?

**Comment by spacewander** [comment]
I am quite annoyed by 
https://github.com/apache/apisix/pull/4476
https://github.com/apache/apisix/pull/4477

Feel free to reopen it if you find a way to add mark without:
1. adding visible mark on the title
2. breaking the cross-reference among markdowns

File: docs/en/latest/getting-started.md (hunk 1 above)
Comment (iamayushdas) [reply docs/en/latest/getting-started.md:24]: @spacewander Sir, first of all I am extremely sorry flooding PRs without discussing,
My PROPOSAL:
Docusaurus has opted this way for adding id's to be more precise. they have changed the md style in such a way that we can add precise id without harming our website
reference md file - https://github.com/facebook/docusaurus/blob/master/website/docs/api/plugins/plugin-content-blog.md
screenshot: 
image


website created from above md file: https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog
screenshot: 
https://docusaurus.io/docs/api/plugins/@docusaurus/plugin-content-blog
Screenshot 2021-06-24 at 3 55 22 PM

now please visit this issue: https://github.com/apache/apisix-website/issues/363

**Pull request closed without merging**